src/client/gameui.cpp

```diff
@@ -116,8 +116,8 @@ void GameUI::update(const RunStats &stats, Client *client, MapDrawControl *draw_
 			<< std::setprecision(1)
 			<< " | view range: "
 			<< (draw_control->range_all ? "All" : itos(draw_control->wanted_range))
-			<< std::setprecision(3)
-			<< " | RTT: " << client->getRTT() << "s";
+			<< std::setprecision(2)
+			<< " | RTT: " << (client->getRTT()*1000.0) << "ms";
```

Comment:
I might be wrong but, perhaps this should be `/ 1000.0` if getRTT returns seconds?

Also, please state floats with an `f`:
`1000.0f`

Reply:
And, spaces either side of `*` are needed.